package.json

```diff
@@ -11,6 +11,7 @@
     "test": "jest",
     "test:watch": "jest --watch",
     "start": "webpack --config webpack.dev.js",
```

Comment:
There's probably not a real need for both `start` and `start:dev` without Sinatra, so you could probably remove this line.

Reply:
```suggestion
```

Reply:
Ok I'll just rename the current `start:dev` to `start` then.